src/main/java/com/google/devtools/build/lib/remote/SimpleBlobStoreFactory.java

```diff
@@ -72,8 +72,10 @@ public static boolean isRemoteCacheOptions(RemoteOptions options) {
   }
 
   private static SimpleBlobStore createHttp(RemoteOptions options, Credentials creds) {
+    Preconditions.checkNotNull(options.remoteCache, "remoteCache");
+    Preconditions.checkArgument(options.remoteCache.startsWith("http"), "remoteCache should start with http");
```

Comment:
We should handle malformed user input gracefully. Throwing a runtime exception crashes the server. Consider throwing  a `UserExecException` instead.

Reply:
Actually, this is more like 'this should never happen thing'. Before calling a `createHttp` method we do always check `isHttpUrlOptions`, so I'd say this is not for you the user, but for the future developers.

Reply:
oh gotcha. done.

Reply:
added to lower case